### ISSUE TITLE
Integration tests failing

### ISSUE THREAD

**Issue by ddriddle** [techservicesillinois/awscli-login#203]
The integration Tests (POSIX) are failing:

```
Run make integration-tests
docker-compose up -d
make: docker-compose: No such file or directory
make: *** [Makefile:107: .idp.docker] Error [12](https://github.com/techservicesillinois/awscli-login/actions/runs/10532232313/job/29185995020#step:13:13)7
Error: Process completed with exit code 2.
```

**Comment by ddriddle** [comment]
It looks like docker compose V1 has been deprecated. We need to upgrade to V2:

https://github.com/orgs/community/discussions/116610

**Comment by ddriddle** [comment]
Addressed by #204.
